riptide-stream/README.md

```diff
@@ -1,7 +1,7 @@
 # Riptide: Streams Extension
```

Comment:
Could document that this follows a slightly more liberal interpretation of the `json-seq` spec, as it will ignore record separators anywhere in the content instead of requiring them between separate objects.

The `x-json-stream` type does not have a definite specification, but I would understand it as json values separated by optional newlines.